docs/user_guides/snapshot_user_guide.md

```
@@ -0,0 +1,716 @@
+
+err = client.CreateSnapshot(context.Background(), createOpt)
+```
+
+Parameters:
+- snapshot_name (string): User-defined unique name for the snapshot
+- collection_name (string): Name of the collection to snapshot
+- description (string, optional): Description of the snapshot
+
+### List Snapshots
+
+List existing snapshots for collections.
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)